# note I is rigid body frame
R_ItoC: [-0.026,-1.000,0.008, -0.387,0.002,-0.922, 0.922,-0.027,-0.387]
p_IinC: [-0.034,-0.062,-0.163]